{{cookiecutter.project_slug}}/k8s/templates/secrets.yaml.template

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+data:
```

If we change the key to `stringData`, it doesn't need to be base64 encoded:
```suggestion
stringData:
```